system/jlib/jsocket.hpp

```diff
@@ -140,11 +140,11 @@ extern jlib_decl IpAddress &localHostToNIC(IpAddress &ip);
 class jlib_decl SocketEndpoint : extends IpAddress
 {
 public:
-    SocketEndpoint()                                            { set(NULL,0); };
+    SocketEndpoint() = default;
     SocketEndpoint(const char *name,unsigned short _port=0)     { set(name,_port); };
     SocketEndpoint(unsigned short _port)                        { setLocalHost(_port); };
     SocketEndpoint(unsigned short _port, const IpAddress & _ip) { set(_port,_ip); };          
-    SocketEndpoint(const SocketEndpoint &other) : IpAddress(other) { port = other.port; }
+    SocketEndpoint(const SocketEndpoint &other) = default;
```

Comment:
Without this there'd be a default copy constructor implicitly defined I think, so this line not strictly necessary?
But probably nice to declare anyway to show it is expected.

Reply:
I think it probably would be, but I wanted to play it safe..